### PR TITLE
Show ETA in status message

### PR DESCRIPTION
**Pull request by PainKiller3**
Signed-off-by: PainKiller3 <ninadpatil100@gmail.com>

### CODE REVIEW

File: src/download_tools/utils.ts [hunk 1]
```diff
@@ -113,6 +113,23 @@ export function findAriaFilePath(files: any[]): string {
  * @param {any[]} files The list of files in the download
  * @returns {StatusMessage} An object containing a printable status message and the file name
  */
+
+function timeLeft (totalLength: number, speed: number) {
+ var time = (totalLength/1048576)/((speed/125000)/8);
```
Comment (out386) [inline src/download_tools/utils.ts]: Why use different factors to scale the speed and length? Why scale them at all? Length should be in B, and speeds should be in B/s, so there will be no scaling required.

Comment (Genos2000) [reply src/download_tools/utils.ts]: cz it works xD

Comment (PainKiller3) [reply src/download_tools/utils.ts]: czwhynot xD
also heroku was giving some errors so needed that otherwise it not works

Comment (destroyer19991) [reply src/download_tools/utils.ts]: verified it works xD

Comment (akhilnarang) [reply src/download_tools/utils.ts]: Even `sudo rm -rf / --no-preserve-root` works. Doesn't mean it should be used.

Comment (out386) [reply src/download_tools/utils.ts]: > Even `sudo rm -rf / --no-preserve-root` works. Doesn't mean it should be used.

This.

**Comment by out386** [comment]
Adapted and merged in 714bcb59ced2aa4c3d72856c210a5de034f45627

**Pull request closed without merging**